freqtrade/strategy/interface.py

```diff
@@ -535,9 +556,12 @@ def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool,
                 and current_profit <= ask_strategy.get('sell_profit_offset', 0)):
             # sell_profit_only and profit doesn't reach the offset - ignore sell signal
             sell_signal = False
-        else:
-            sell_signal = sell and not buy and ask_strategy.get('use_sell_signal', True)
+        elif ask_strategy.get('use_sell_signal', True):
+            sell = sell or self.custom_sell(trade.pair, trade, date, current_rate, current_profit)
```

Comment:
this should be wrapped in `strategy_safe_wrapper` ... it'll catch potential user exceptions - which i see a necessity for these functions, as they might get quite nested, where not every branch might work in some cases.